docs/source/users/index.md

```diff
@@ -87,6 +87,12 @@ To install the JupyterLab extension, you can run:
 pip install jupyter-ai
 ```
 
+In some cases, subsidiary packages to be installed, e.g., required by some model providers and some file formats. To ensure such cases are handled, to install every dependency to get access to all models currently supported by `jupyter-ai`, you should run (and then reload Jupyter Lab):
```

Comment:
```suggestion
In some cases, subsidiary packages need to be installed, e.g., because they are required by some model providers and some file formats.
To ensure such cases are handled you can install every dependency which will give you access to all models currently supported by `jupyter-ai`
To install every dependency, run (and then restart JupyterLab):
```

The sentences appear incomplete. I am not sure how to make it sound best, please do adjust the suggestions as you seem it fit best.

Reply:
Also "subsidiary packages" seem very technical, not sure if "third-party" would be better here?

Reply:
@krassowski - Thanks for the helpful comments! I have implemented these.